release_notes/ocp-4-10-release-notes.adoc

```diff
@@ -2972,6 +2972,12 @@ Space precluded documenting all of the container images for this release in the
 
 link:https://access.redhat.com/solutions/6962052[{product-title} 4.10.17 container image list]
 
+[id="ocp-4-10-17-bug-fixes"]
+==== Bug fixes
+* Previously, the federation endpoint for Prometheus was not exposed. Thus, you could not access it to scrape user-defined monitoring metrics from a network location outside the cluster.
```

Comment:
```suggestion
* Previously, the federation endpoint for Prometheus storing user-defined metrics was not exposed. Thus, you could not access it to scrape user-defined monitoring metrics from a network location outside the cluster.
```
Just a minor addition to add a detail. in 4.10 we do expose `/federate` but for the in-cluster Prometheus instance. This adds the same feature for the user-workload Prometheus instance.

Reply:
fixed